### DISCUSSION TITLE
Operator deployment in different namespace than kube-system.

### DISCUSSION THREAD

**Discussion by illarios** [microsoft/retina#487] [Q&A] [answered]
As retina is able to be deployed in other namespaces and I can see that the retina-operator-config gets its namespace from the values file:

https://github.com/microsoft/retina/blob/316ad80b4829a464ed60daa50459df4ec27aab92/deploy/manifests/controller/helm/retina/templates/operator.yaml#L258C1-L263C37

I would suggest the same to happen with the operator itself, as the case now is that if you choose another namespace for deployment of retina the operator cannot start without its configuration which apparently exists in the different namespace specified in the values file.

You can see here that the operator's deployment namespace is not configurable:

https://github.com/microsoft/retina/blob/316ad80b4829a464ed60daa50459df4ec27aab92/deploy/manifests/controller/helm/retina/templates/operator.yaml#L6

**Comment by rbtr** [comment]
I'll create an issue out of this - it would be pretty trivial to inject the namespace on all the helm objects just like [this](https://github.com/microsoft/retina/blob/316ad80b4829a464ed60daa50459df4ec27aab92/deploy/manifests/controller/helm/retina/templates/operator.yaml#L263).

**Comment by SRodi** [comment]
@BeegiiK

**Comment by BeegiiK** [comment, marked as answer]
Hey @illarios,

The Retina operator has been updated to support deployment in any specified namespace, as defined in the corresponding values file. This improvement will be available in the next release.
